### DISCUSSION TITLE
Salmen's tasks

### DISCUSSION THREAD

**Discussion by Rahma090** [Team-5-Pinterest-clone/Team-5-Pinterest-clone#34] [General]
1-login page
2- Search component (2)
3- One post comment design
4- Share buttons & comment functionality in One post component
5-Remove "Brand" input in create post component ====> Done

**Comment by Rahma090** [comment]
1-login page =====> Done
2- Search component (2)
3- One post comment design
4- Share buttons & comment functionality in One post component
5-Remove "Brand" input in create post component ====> Done

**Reply by salmenkhelifi1** [reply]
One post comment design done 
  4- Share buttons & comment functionality in One post component done 
  2- Search component (2) in progress

**Reply by salmenkhelifi1** [reply]
![Uploading Screenshot 2023-12-16 231718.png…]()
